.vscode/quarto.code-snippets

```diff
@@ -0,0 +1,139 @@
+{
+	// Place your snippets for Quarto here. 
+	// Each snippet is defined under a snippet name and has a prefix, body and description. 
+	// The prefix is what is used to trigger the snippet and the body will be expanded and inserted. 
+	// Possible variables are:
+	//
+	// - $1 and $2 for tab stops, 
+	// - $0 for the final cursor position, and 
+	// - ${1:label}, ${2:another} for placeholders
+	// 
+	// Placeholders with the same ids are connected.
+	//
+	// Don't forget to edit VS Code to accept tab completion for Quarto - this is not available through the standard settings for Quarto
+	// Go to VS Code settings (Click gear icon in bottom left corner or use the shortcut Cmd+,) and search for: @lang:quarto snippets
+	// Set "Tab Completion" to "On"
+	// Ensure that "Show Snippets" is ticked
+	// Optional: Choose where the snippets comes in the list of suggestions in the field "Editor: Snippet Suggestions"
+	// 
+	// How to use the code snippets:
+	// After you have set the above settings, a suggestion for one of the code snippets below will pop up automatically, when
+	// you start writing the prefix. Press "Enter" or "Tab" to insert the code snippet.
+
+	"Insert TODO formatting":{
+		"prefix": "TODO",
+		"body": [
+			"<!--TODO ${0:Write text here}-->"
+		],
+		"description": "Insert TODO formatting"
+	}
+	
+	"Insert bash formatted text":{
+		"prefix": "bash",
+		"body": [
+			"``` bash"
+			"${0:Write text here}"
+			"```"
+		],
+		"description": "Insert bash formatted text"
+	}
+	
+	"Insert title section":{
+		"prefix": "ttl_sec",
+		"body": [
+			"---"
+			"title: \"\""
+			"description: \"Our reasons for ...\""
+			"author: \"\""
+			"date: last-modified"
+			"categories:"
+			"---"
+		],
+		"description": "Insert title section"
+	}
+
+	"Insert a hidden comment section":{
+		"prefix": "hddn",
+		"body": [
+			"::: content-hidden" 
+			"${0:Write comments here}"
+			":::"
+		],
+		"description": "Insert a hidden content section"
+	}
+
+	"Insert a 2 col table": {
+		"prefix": "tbl2",
+		"body": [
+				"|${1:title} |${0:title} |",
+				"| --- | --- |",
+				"|   |   |"
+		],
+		"description": "Insert a 2 col table"
+	}    
+
+	"Insert a 3 col table": {
+		"prefix": "tbl3",
+		"body": [
+				"|${1:title} |${2:title} |${0:title} |",
+				"| --- | --- | --- |",
+				"|   |   |   |"
+		],
+		"description": "Insert a 3 col table"
+	}
+
+// All our key words are included in this code snippet
```

Comment:
I'm not sure this is the right place for this. I don't yet know how or if it is important to have an "official list" of category tags or if we simply run a quick check every so often to make sure there aren't tags that are too similar. But for now, until we think of other ways of doing this, this is fine :relaxed: :mechanical_arm: